### PR TITLE
Add flag to query logs by client ID.

### LINKED ISSUE #131: Support filtering by app when tailing logs [CLI-61]
`auth0 logs --clientId xyz` would filter logs for that particular app. This is useful for troubleshooting to avoid all the noise of other apps.

### PR DESCRIPTION
**Pull request by chrisscott**
### Description

Add a flag to the `logs` command to query by client ID.

### References

Fixes #131 

### Testing

`auth0 logs -c [application client id]`

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [X] All active GitHub checks for tests, formatting, and security are passing
- [X] The correct base branch is being used, if not `master`

### CODE REVIEW

File: internal/cli/logs.go [hunk 1]
```diff
@@ -68,12 +74,18 @@ Show the tenant logs.
...
+						queryParams := []management.RequestOption{
 							management.Query(fmt.Sprintf("log_id:[%s TO *]", lastLogID)),
 							management.Parameter("page", "0"),
 							management.Parameter("per_page", "100"),
 							management.Parameter("sort", "date:-1"),
-						)
+						}
+
+						if flags.ClientID != "" {
+							queryParams = append(queryParams, management.Query(fmt.Sprintf(`client_id:"%s"`, flags.ClientID)))
```
[the remark this answers is not in the document]
Comment (chrisscott) [reply internal/cli/logs.go]: Good call. Updated.

File: internal/cli/logs.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ import (
 	"gopkg.in/auth0.v5/management"
 )
 
-func getLatestLogs(cli *cli, n int) ([]*management.Log, error) {
+func getLatestLogs(cli *cli, n int, ClientID string) ([]*management.Log, error) {
```
Comment (Widcket) [inline internal/cli/logs.go:13]:
```suggestion
func getLatestLogs(cli *cli, n int, clientID string) ([]*management.Log, error) {
```

File: internal/cli/logs.go [hunk 3]
```diff
@@ -20,11 +20,16 @@ func getLatestLogs(cli *cli, n int) ([]*management.Log, error) {
...
 
-	return cli.api.Log.List(
+	queryParams := []management.RequestOption{
 		management.Parameter("sort", "date:-1"),
 		management.Parameter("page", fmt.Sprintf("%d", page)),
-		management.Parameter("per_page", fmt.Sprintf("%d", perPage)),
-	)
+		management.Parameter("per_page", fmt.Sprintf("%d", perPage))}
+
+	if ClientID != "" {
```
Comment (Widcket) [inline internal/cli/logs.go:28]:
```suggestion
	if clientID != "" {
```

File: internal/cli/logs.go [hunk 4]
```diff
@@ -20,11 +20,16 @@ func getLatestLogs(cli *cli, n int) ([]*management.Log, error) {
...
+		queryParams = append(queryParams, management.Query(fmt.Sprintf(`client_id:"%s"`, ClientID)))
```
Comment (Widcket) [inline internal/cli/logs.go:29]:
```suggestion
		queryParams = append(queryParams, management.Query(fmt.Sprintf(`client_id:"%s"`, clientID)))
```

**Review by Widcket — APPROVED** [review]
A couple of nits, other than that good work! Thanks for your contribution.

**Pull request merged**